modules/cassandra/cassandra.go

```diff
@@ -126,3 +145,11 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 
 	return c, nil
 }
+
+// TLSConfig returns the TLS configuration for the Cassandra container, nil if TLS is not enabled.
+func (c *CassandraContainer) TLSConfig() *tls.Config {
```

Comment:
@mdelapenya quite a common pattern in modules, we should consider returning an error instead.

Reply:
You mean the pointer to the config and the error, right? I think we changed the tlscert library precisely for that, so good to me to change it

Reply:
Oh I see, the recently added methods for Redis and Valkey. For Valkey, we are in the same release cycle, we can change it without BC, for Redis, it's a BC